Casks/r/rar.rb

```diff
@@ -12,7 +12,7 @@
 
   livecheck do
     url "https://www.rarlab.com/download.htm"
-    regex(/>s*RAR\s*for\s*macOS.*v?(\d+(:?\.\d+)+)[ "<]/i)
+    regex(/>s*RAR\s*for\s*macOS.*v?(\d+(:?\.\d+)+)["<]/i)
```

Comment:
```suggestion
    regex(/>\s*RAR\s+for\s+macOS.*?v?(\d+(:?\.\d+)+)\s*</i)
```

It seems the first `s` was meant to be whitespace, i.e., `\s`. Since the version is inside a `<b></b>` tag, we can update the regex to use these as boundaries.